database/access_rules.go

```diff
@@ -341,3 +345,197 @@ func (c *UtilityClient) WaitForAccessRuleDeleted(input *GetAccessRuleInput, poll
 	})
 	return info, err
 }
+
+type DefaultAccessRuleInfo struct {
+	// Name of the DBaaS service instance.
+	ServiceInstanceID string `json:"-"`
+	// Enabled for every DB Service Instance
+	EnableSSH *bool
+	// Single Instance Rules
+	EnableHTTP       *bool
+	EnableHTTPSSL    *bool
+	EnableDBConsole  *bool
+	EnableDBExpress  *bool
+	EnableDBListener *bool
+	// RAC Rules
+	EnableEMConsole     *bool
+	EnableRACDBListener *bool
+	EnableScanListener  *bool
+	EnableRACOns        *bool
+}
+
+// Default Access Rule prefixes
+var DefaultAccessRuleNames = map[string]string{
+	"EnableSSH":           "ora_p2_ssh",
+	"EnableHTTP":          "ora_p2_http",
+	"EnableHTTPSSL":       "ora_p2_httpssl",
+	"EnableDBConsole":     "ora_p2_dbconsole",
+	"EnableDBExpress":     "ora_p2_dbexpress",
+	"EnableDBListener":    "ora_p2_dblistener",
+	"EnableEMConsole":     "ora_p2_emconsole",
+	"EnableRACDBListener": "ora_p2_db_listener",
+	"EnableScanListener":  "ora_p2_scan_listener",
+	"EnableRACOns":        "ora_p2_ons",
```

Comment:
_minor_ we could probably make these keys constants rather than magic strings all over the place, but no biggie.